### ISSUE TITLE
Using `edit_physio()` in Spyder breaks console after closing interactive plot session.

### ISSUE THREAD

**Issue by smoia** [physiopy/prep4phys#13]
This happens in Spyder 4 (didn't test on previous Spyder versions), in Ubuntu 18.04 and latest winpython.
If we can figure out why, it would be great - starting with trying to reproduce the problem in different environments @physiopy/all .

**Comment by me-pic** [comment]
Revisiting that issue: The interactive plot is working for me in jupyter notebook when I specified either `%matplotlib widget` or `%matplotlib ipympl`. I can delete and add peaks. However, the deleted/added peaks do not appear afterward in the history of the Physio object...

**Comment by me-pic** [comment]
Yes. If I delete some peaks, nor the peaks when I call peaks on the object nor the history are updated..

**Comment by smoia** [comment]
See: https://github.com/matplotlib/matplotlib/issues/13302

**Comment by me-pic** [comment]
Update: it works when using `%matplotlib qt`:

- [X]  Interacting with the figure (i.e. zoom in, moving across it)
- [X] Deleting / adding peaks / troughs
- [X] Updating the peaks/troughs list (i.e., `data.peaks`) and history
